zerver/tests/test_user_groups.py

```diff
@@ -1452,15 +1453,15 @@ def test_user_group_deactivation_with_group_used_for_settings(self) -> None:
             support_group.deactivated = False
             support_group.save()
 
+            # Unarchive the stream for the next test
+            do_unarchive_stream(stream, "support", acting_user=None)
```

Comment:
This change was needed so that stream is reactivated before its `can_send_message_group` setting is changed as a message is sent on changing that setting and we do not allow sending message in a deactivated stream.